PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/core/FunctionUtils.java

```diff
@@ -147,8 +150,19 @@ private static void copyFilesWithDefaultContent(Path sourcePath, File dest, Stri
         }
     }
 
+    private static void updateLocalSettingValues(File target, Map<String, String> appSettings) throws IOException {
+        final Gson gson = new GsonBuilder().setPrettyPrinting().disableHtmlEscaping().create();
```

Comment:
add an writeJsonToFile method in a new JsonUtility class

Reply:
Done